doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCertificateStoreProvider.xml

```diff
@@ -26,7 +26,7 @@
             <param name="masterKeyPath">The master key path.</param>
             <param name="encryptionAlgorithm">The encryption algorithm. Currently, the only valid value is: <c>RSA_OAEP</c>
             </param>
-            <param name="columnEncryptionKey">The encrypted column encryption key.</param>
+            <param name="columnEncryptionKey">The plaintext column encryption key.</param>
```

Comment:
When I first read the old line, I interpreted "encrypted" to apply to the column. In other words, "The encryption key for the encrypted column". If we are clarifying that the key is plaintext, perhaps we could say:
```suggestion
            <param name="columnEncryptionKey">The plaintext encryption key for the encrypted column.</param>
```

How do the ODBC and JDBC drivers phrase it?

Reply:
Looking at the other driver docs, I rescind my suggestion. The current seems appropriate in the context of the other methods and other driver descriptions.